packages/dds/merge-tree/src/test/snapshot.utils.ts

```diff
@@ -61,6 +61,19 @@ export class TestString {
 		this.insert(this.client.getLength(), text, increaseMsn);
 	}
 
+	public insertMarker(pos: number, increaseMsn: boolean) {
```

Comment:
New marker code ended up not being necessary, but it seems useful to leave around. Will delete if asked.